src/main/java/spoon/support/DerivedProperty.java

```diff
@@ -23,10 +23,16 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-/** Tells that a metamodel property is derived, ie computed from the value of another property.
+/**
+ * Tells that a metamodel property is derived, ie computed from the value of another property.
+ *
  * For instance {@link CtType#getFields()}  is derived from {@link CtType#getTypeMembers()}
  *
- * This annotation is used for specifying CtScanner: derived properties are never scanned.
+ * It can be put on getter ond setters.
+ *
+ * Contracts:
+ * - A setter with @DerivedProperty never triggers a model intercession event.
```

Comment:
> intercession event

Do you mean something like this?
```java
class CtConstructorCallImpl
...
	@DerivedProperty
	public <T extends CtActualTypeContainer> T addActualTypeArgument(CtTypeReference<?> actualTypeArgument) {
		if (getExecutable() != null) {
			getExecutable().addActualTypeArgument(actualTypeArgument);
		}
		return (T) this;
	}
```
It triggers the event, but not directly on the `this` node, but on some delegate? Did I understood it well? Improve description?

Is there a test for setters which does not start with `set` prefix, but with add/remove?